### ISSUE TITLE
Schedule VI Balance Sheet

### ISSUE THREAD

**Issue by harshit-30** [frappe/erpnext#28226]
### Problem Statement

Schedule VI was a requirement for many years as per company law but this has changed to Schedule III now. 

But Schedule VI is a good way to view the Balance Sheet and would definitely give an edge if the ERP can throw up the Balance Sheet in Schedule VI/III format. This would help the entity using the ERP to skip one level of generating the TB and then doing the Schedule VI/III in xl.

Basically, all entities have to prepare the Sch VI/III format of financials for audit and sign off. So what we generally do is take the TB in XL and come up with a mapping to the Sch VI/III format in XL. Hence if the system itself can do this mapping and generate the Sch VI/III balance sheet, P&L and schedules it would be a major value add.

**Comment by grynn-in** [comment]
Is it a country specific request? Would be great to tag it as such as well!

**Comment by DaiyaAkshay** [comment]
Hi, any update on this request. 
I really need this Schedule VI/III feature for one of my clients.

**Comment by gb-123-git** [comment]
+1

**Comment by singhranbir** [comment]
+1
